# Assignment 3 - Z-Test 

## Null and Alternative Hypothesis: 

_H$0$_: New route >= 36 +-6 min.

_H$1$_: New route < 36 +- 6 min. 

I will set my significance level at an alpha of .05

In [7]:
import numpy as np
import os
import pandas as pd

In [27]:
def z(m_new, n_new, mu=36, sigma=6): 
    return (mu-m_new) / (sigma / np.sqrt(n_new))

In [28]:
df = pd.read_csv('/Users/katharinevoorhees/puidata/times.txt',header=None)
 

In [29]:
df.head()

0
0  31.622239
1  32.821376
2  30.229101
3  31.413766
4  39.010550

In [37]:
print ("The z-score is: ", z(df.mean(),df.count()) )

The z-score is:  0    2.556397
dtype: float64


We set our a at .05, or a 95% confidence that the difference in our sample mean is due to a true difference and not just chance. This means that in order to reject the null hypothesis we were looking for a Z score of 1.96 or higher. Since 2.556 is higher than 1.96 **we reject the null hypothesis in this case.** 